config/secrets.yml

```diff
@@ -17,4 +17,4 @@ test:
   secret_key_base: ee4280f086c62260dbd973a322a84720db40913555b2ee165b04b02a0ba4907b39d74d0819f15913d8c6ce1b80c91be12f181322474ea150a9d425619db0101a
 
 production:
-  secret_key_base: <%= ENV["askup_secret_key_base"] %>
```

Comment:
The secret_key_base should not be hardcoded for the production version of the site. Just change it back to what it used to be.